### PR TITLE
Adding a feature flag for facility locator front end

### PR DESCRIPTION
**Pull request by darkbushido**
the flag will toggle how we display covid19 information

## Description of change

Adding a feature flag for the front end.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#17908

## Things to know about this PR
* `facility_locator_lighthouse_covid_vaccine_query`

### CODE REVIEW

**Comment by brangi** [comment]
FE implementation - https://github.com/department-of-veterans-affairs/vets-website/pull/15502/files

File: config/features.yml [hunk 1]
```diff
@@ -57,6 +57,10 @@ features:
     actor_type: user
     enable_in_development: true
     description: A fast and dirty way to get the operating status from lighthouse
+  facility_locator_lighthouse_covid_vaccine_query:
```
Comment (erluti) [inline config/features.yml:60]: looks like FE is looking for `facilityLighthouseCovidVaccineQuery` which doesn't match with this, does it?

Comment (brangi) [reply config/features.yml:60]: This should fix it - https://github.com/department-of-veterans-affairs/vets-website/pull/15502/commits/3cfc68017ce90b0e330dfdc6ddc57749753d3152

**Review by erluti — APPROVED** [review]
LGTM

File: config/features.yml (hunk 1 above)
Comment (erluti) [reply config/features.yml:60]: Thanks for the update!

**Pull request merged**